fdbserver/EncryptKeyProxyInterface.h

```diff
@@ -142,6 +143,7 @@ struct EKPGetLatestBaseCipherKeysReply {
 	Arena arena;
 	std::vector<EKPBaseCipherDetails> baseCipherDetails;
 	int numHits;
+	Optional<UID> debuId;
```

Comment:
```suggestion
	Optional<UID> debugId;
```

Reply:
Thanks for pointing. 
Actually I found that _debugId_ was not getting serialized for the same struct. Shall audit all others as well, just incase. 

Thanks

Reply:
Sorry took sometime to page-in all details, actually we don't nee this field at all. Will remove it.